QSB/AnglerFish/AnglerManager.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using OWML.Common;
+using QSB.AnglerFish.WorldObjects;
+using QSB.Player;
+using QSB.Utility;
+using QSB.WorldSync;
+
+namespace QSB.AnglerFish {
```

Comment:
please keep code style consistent with rest of qsb
- tabs instead of spaces
- { on new lines
- always having braces on single line ifs
etc

Reply:
done

Reply:
i cant see any changes...?